wordpress-mobile/WordPress-Android
# Remove the Gutenberg dialog on new sites

Remove the Gutenberg dialog that was showning after creating a new site. 
Also change the default for freshly added self hosted sites.

Test 1:
- Create a new site.
- Create a new post, make sure the block editor is showing and there is no dialog about it.

Test 2:
- Create a new user.
- Create a new site.
- Create a new post, make sure the block editor is showing and there is no dialog about it.

Test 3:
- Add a self hosted site.
- Create a new post, make sure the block editor is showing and there is no dialog about it.

PR submission checklist:
